wireshark_x86.sls

```diff
@@ -1,4 +1,13 @@
 wireshark_x86:
+  '1.12.8':
+    full_name:  'Wireshark 1.12.8'
+    installer: 'http://wiresharkdownloads.riverbed.com/wireshark/win32/Wireshark-win32-1.12.6.exe'
```

Comment:
@holmboe thanks for the report. fixed in https://github.com/saltstack/salt-winrepo/pull/655